RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py

```diff
@@ -30,3 +30,20 @@
                                                        872420273,872436907,872420147,872439731,
                                                        872436657,872420397,872439732,872439339,
                                                        872439603])
+
+from Configuration.Eras.Modifier_run2_miniAOD_devel_cff import run2_miniAOD_devel
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv1_cff import run2_nanoAOD_94XMiniAODv1
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv2_cff import run2_nanoAOD_94XMiniAODv2
+from Configuration.Eras.Modifier_run2_nanoAOD_102Xv1_cff import run2_nanoAOD_102Xv1
+for modifier in run2_miniAOD_devel, run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94XMiniAODv2, run2_nanoAOD_102Xv1:
+    modifier.toModify(ecalBadCalibFilter, baddetEcal = [872439604,872422825,872420274,872423218,
+                                                        872423215,872416066,872435036,872439336,
+                                                        872420273,872436907,872420147,872439731,
+                                                        872436657,872420397,872439732,872439339,
+                                                        872439603,872422436,872439861,872437051,
+                                                        872437052,872420649,872421950,872437185,
+                                                        872422564,872421566,872421695,872421955,
+                                                        872421567,872437184,872421951,872421694,
+                                                        872437056,872437057,872437313])
+for modifier in run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94XMiniAODv2, run2_nanoAOD_102Xv1:
+    modifier.toModify(ecalBadCalibFilter, EcalRecHitSource = cms.InputTag('reducedEgamma','reducedEERecHits'))
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Reply:
Do you mean `EcalRecHitSource = ('reducedEgamma','reducedEERecHits')`?

Reply:
right.
IIUC, ```'reducedEgamma:reducedEERecHits'``` should work as well